src/frontends/paddle/tests/CMakeLists.txt

```diff
@@ -75,6 +89,20 @@ if(paddlepaddle_FOUND AND PADDLEDET_RESULT)
             DESTINATION tests/${TEST_PADDLE_MODELS_DIRNAME}
             COMPONENT tests
             EXCLUDE_FROM_ALL)
+
+    execute_process(
+        COMMAND ${Python3_EXECUTABLE} -c "import paddle; print(paddle.__version__)"
+        OUTPUT_VARIABLE PADDLE_VERSION
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+    message(STATUS "PaddlePaddle version: ${PADDLE_VERSION}")
+
+    if(PADDLE_VERSION VERSION_GREATER_EQUAL "2.6.0" OR PADDLE_VERSION VERSION_EQUAL "0.0.0")
```

Comment:
Instead of overwrite the content in ops.py, I think it's better to update the download information of ops.py file in Line 50-52 if you can find the new one

Reply:
I tried to find a suitable version. However I don't find one. When Paddledetection removed paddle.fluid it also removed many apis such as `roi_pool`, `roi_align` and so on. 

Some ops like roi_pool even removed from paddle.

Reply:
I find roi_pool in Paddle pypthon/paddle/vision/ops.py. Maybe some missing ops can all be found there. Can you use the latest ops.py and change the new api in Paddle for the missing ops?